server/util/disk/disk_windows.go

```diff
@@ -10,6 +10,9 @@ import (
 	"golang.org/x/sys/windows"
 )
 
+// 4KiB block size
```

Comment:
How about adding the reference links directly to this comment (for NTFS and ReFS)?